ceph.spec.in

```diff
@@ -1304,41 +1304,45 @@ if [ $FIRST_ARG -ge 1 ] ; then
 fi
 
 %files mgr
+%{_bindir}/ceph-crash
```

Comment:
I think this belongs in ceph-base, which is required by all of the daemon packages (osd mds mgr mon), since they can all crash and leave dumps around.